docs/deploying/vps.md

```diff
@@ -0,0 +1,146 @@
+      CHECK_ORIGIN=false
+      PORT=5002
+      VIRTUAL_HOST=localhost
+      EOF
+    - [
+        su,
+        jellyfish,
+        -s,
+        /bin/bash,
+        -c,
+        "docker run -d \
+        --restart unless-stopped \
+        -p 50000-50050:50000-50050/udp \
+        -p 80:5002 \
+        --env-file /opt/jellyfish/env-file \
+        -v /opt/jellyfish/jellyfish_output:/app/jellyfish_output \
+        ghcr.io/jellyfish-dev/jellyfish:$JELLYFISH_VERSION",
+      ]
+  ```
+</details>
+
+You'll need to take care of two lines in that template:
+
+- `- export JELLYFISH_VERSION=0.1.0-ed317b`. Change value if you want to deploy different version of jellyfish. You can find list of available versions in [our package repository](https://github.com/jellyfish-dev/jellyfish/pkgs/container/jellyfish)
+
+- `SERVER_API_TOKEN=test_token` Server API token is a token you'll need to connect to Jellyfish via API or SDK. Write here a chosen secret value you're going to remember later.
+
+Copy the file and change up those two variables. You're going to need a content of the file in the next section.
+
+### Other tools we'll install with _cloud-init_
+
+As you might noticed we've chosen some packages to be installed:
+
+- `ufw` - Uncomplicated Firewall. We'll need it to allow or block trafic on specific ports [https://help.ubuntu.com/community/UFW](https://help.ubuntu.com/community/UFW).
+- fail2ban - Tool to block traffic recognized as unauthorized [https://www.fail2ban.org/wiki/index.php/Main_Page](https://www.fail2ban.org/wiki/index.php/Main_Page)
+- `gzip` - Compression Utility [https://www.gzip.org/](https://www.gzip.org/)
+- `containerd.io`, `docker-ce`, `docker-ce-cli` - Docker and Docker related libraries.
+
+## Setting up Hetzner cloud VPS with _cloud-config_
+
+Now, we're going to use a content of the file you prepared in Hetzner cloud VPS creating form. To do it scroll to the _Cloud config_ section of the form and paste there a content of your _cloud-config.yaml_ file.
+
+![Hetzner Cloud - placing cloud-config](/img/vps-2.png)
+
+:::caution
+_cloud-init_ will recognize the file as it's configuration only if the file starts with that comment:
+```
+#cloud-config
+...
+```
+So be carefull while copying.
+:::
+
+That's it. You can click 'Create & Buy' an a server will start to configurate.
+
+## Checking _cloud-init_ status
+
+Configuring your server will takes some time, but it's available for you almost immediately after creation. You can log into server using _ssh_:
+```
+ssh root@<VPS_IP>
+```
+then, in server's terminal you can check current status of cloud init:
+
+```bash
+cloud-init status
+```
+
+Posible responses are `running`, `error` and `done`. The first one informs you that _cloud-init_ is still configuring you server. Second one means that something whent wrong. You can find logs from the _cloud-init_ run in `/var/log/cloud-init-output.log` file. `done` means that your jellyfish server is ready to be used.
+
+
```

Comment:
I think you could mention at the end that we can use Jellyfish Dashboard (e.g. the online hosted version) to check if the Jellyfish is working correctly.

Reply:
I've added additional paragraph about it